### ISSUE TITLE
bin/grav cli unknown CacheCommand class error

### ISSUE THREAD

**Issue by nex84** [getgrav/grav#2442]
Hello,

Since the 1.6.2 upgrade, the **bin/grav** cli doesn't work with the following error :
```
PHP Fatal error:  Uncaught Error: Class 'Grav\Console\Cli\CacheCommand' not found in /var/www/html/bin/grav:43
Stack trace:
#0 {main}
  thrown in /var/www/html/bin/grav on line 43
```

I may be wrong but it seems to be related to commit 7762f0c85eaf4cfbb5b9a4d993b5cde1274cd17c

**Comment by rhukster** [comment]
It’s normal and a one time message. It does not effect the upgrade. 

I had changed this class name in 1.6 but it turns out it gave a much worse error for people not already on 1.6 when upgrading. The functionality has not changed nor the actual command.

**Comment by garytube** [comment]
mhh, I tried to use the bin/grav CLI tools but it fails.

`bin/grav
PHP Fatal error:  Uncaught Error: Class 'Grav\Console\Cli\CacheCommand' not found in .../bin/grav:43
Stack trace:
#0 {main}
  thrown in.../bin/grav on line 43`

I upgraded from 1.6.0 rc4 to 1.6.2 

am I missing something

**Comment by rhukster** [comment]
If you scroll up I think you willl find the upgrade **was** successful, but cache clear was not.

**Comment by garytube** [comment]
the upgrade to 1.6.2? that worked fine.. 

im trying to execute bin/grav  by myself to clear the cache manually  - that does not work anymore

**Comment by nex84** [comment]
I upgraded from 1.5.10 to 1.6.2 without any issue. 
The problem is that I can't do a backup using CLI anymore (and probably any other actions using bin/grav).

For example, if I try to launch the following command (as per 1.6 documentation) : **bin/grav backup 1** it fails immediatly and does nothing :
```
$ bin/grav backup 1
PHP Fatal error:  Uncaught Error: Class 'Grav\Console\Cli\CacheCommand' not found in /var/www/html/arcanexus.com/www/bin/grav:43
Stack trace:
#0 {main}
  thrown in /var/www/html/arcanexus.com/www/bin/grav on line 43
```
It happens everytime I try to launch the command, so no backups can be done.

**Comment by rhukster** [comment]
Grr. Looking

**Comment by rhukster** [comment]
dammit, missed that one...

on line 43 in `bin\grav` change it to:

```
new \Grav\Console\Cli\ClearCacheCommand(),
```

**Comment by nex84** [comment]
I suppose that it should be easily fixed by renaming that class in bin/grav code too (line 43), but I don't know if there will be impacts.

**Comment by rhukster** [comment]
Fixed in 1.6.3
